connections/oikotie/oikotie_mapper.py

```diff
@@ -153,7 +153,7 @@ def map_balcony(elastic_apartment: ElasticApartment) -> Optional[Balcony]:
 
 
 def map_living_area(elastic_apartment: ElasticApartment) -> Optional[LivingArea]:
-    if getattr(elastic_apartment, "living_area", None):
+    if getattr(elastic_apartment, "living_area") is not None:
```

Comment:
Now this won't allow `living_area` attribute to be missing from `elastic_apartment`.  If we can rely on that, then there is no reason to use `getattr()` in the first place, and I would change this to use just `elastic_apartment.living_area` instead.

Reply:
That is true. Changed to all fields which were changed.